apps/api/src/utils/AesTools.ts

```
@@ -0,0 +1,73 @@
+class CryptData {
+  constructor(
+    public iv = '',
+    public data = ''
+  ) {}
+}
+
+class AesTools {
+  private static secKey = 'tmppasswordver1.'
+
+  public static setSecretKey(sec: string): void {
+    this.secKey = sec
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)